apps/courses-site/specs/index.spec.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
 import Page from '../src/app/page';
 
 describe('Page', () => {
-  it('should render successfully', () => {
-    const { baseElement } = render(<Page />);
-    expect(baseElement).toBeTruthy();
+  it('should render heading', () => {
+    render(<Page />);
+
+    expect(
+      screen.getByRole('heading', { name: 'Courses' })
+    ).toBeInTheDocument();
```

Comment:
I learned today that `toBeInTheDocument` only checks the DOM not the CSS visible which `toBeVisible` does do, so I would recommend moving to that